examples/record-optional-fields/record_optional_fields.bal

```diff
@@ -25,7 +25,9 @@ public function main() {
 
     // Before accessing/using an optional field, it must be added to the record.
     john.age = 25;
-    io:println("Age: ", john.age);
+    // Optional fields are accessed using `?.` notation on the record.
+    // This returns the value if the field is present in the record, `()` if not.
```

Comment:
```suggestion
    // This returns the value if the field is present in the record. Else, returns `()` if not.
```

Reply:
Do we need both else and if not? Feels redundant IMO.